plugins/index.html

```diff
@@ -17,6 +17,19 @@ <h2>How to configure a plugin</h2>
     ]
 });</code></pre>
 
+  <p>When using NPM, the plugins are installed
+  in <code>node_modules</code> with the package but must be imported or required
+  locally in order to use them:
+    <pre><code class="javascript">
+# ES6: import RegionsPlugin from "wavesurfer.js/dist/plugin/wavesurfer.regions.min.js";
```

Comment:
a hastag isn't a valid comment identifier in javascript, use // instead

Reply:
ah! good idea, sorry!